test/isotest.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+
+const { render } = require("enzyme");
+const React = require("react");
+
+/**
+ * Determines if the passed Component is a React Class or not
+ * `TestUtils.isElement()`
+ * @returns {boolean}
+ */
+function isReactClass(Component) {
+    return typeof Component !== "undefined"
+        && typeof Component.prototype !== "undefined"
+        && typeof Component.prototype.constructor !== "undefined"
+        && typeof Component.prototype.render !== "undefined"
+    ;
+};
+
+/**
+ * Tests that each ComponentClass in Components can be isomorphically rendered on the server.
+ * @param Components        {{ [componentName: string]: any }}              main export from package
+ * @param customProps       {{ [componentName: string]: any}}               custom props per component
+ * @param customChildren    {{ [componentName: string]: React.ReactNode }}  custom children per component
+ */
+module.exports = function isotest(Components, customProps, customChildren) {
```

Comment:
can this be more verbose to make it clear that it _generates_ multiple tests? `generateIsoTests`? same for the import name elsewhere